towhee/tests/mock_operators/norm_data_operator/norm_data_operator.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import NamedTuple
+from towhee.operator import Operator, SharedType
+from numpy import linalg as LA
+import numpy
+
+
+class NormDataOperator(Operator):
+    """
+    Stateful operator
+    """
+
+    def __init__(self, normed_num) -> None:
+        super().__init__()
+        self.normed_num = normed_num
```

Comment:
`normed_num` does not seem to be used in `__call__`. What is this parameter for?

Reply:
It's not used now, because it will show the error `TypeError: 'NoneType' object is not iterable` if __int__ without the argument. Is there another way to solve it?

Reply:
I see, can you open an issue for that and add a `@TODO` here?

Reply:
Of course, there is issue #170.